### PR TITLE
Support installation for Xcode 12.4

### PR DESCRIPTION
**Pull request by devahmedshendy**
### Description
I renamed `selectedPlan(plans)` method to `selectPlan(plans)` to add backward compatibility for Xcode 12.4.

### References
Issue began in this [discussion](https://github.com/swiftyfinch/Rugby/discussions/107), then decision made by @swiftyfinch in this [comment](https://github.com/swiftyfinch/Rugby/discussions/107#discussioncomment-1988851), and a whole discussion opened for it [here](https://github.com/swiftyfinch/Rugby/discussions/115)

### Checklist
- [ ] I've added at least one test that validates that my change is working, if appropriate
- [x] I've followed the code style of the rest of the project
- [x] I've read the [Contribution Guidelines](CONTRIBUTING.md)
- [x] I've updated the documentation if necessary

❤️ Thanks for contributing to the 🏈 Rugby!

### CODE REVIEW

**Comment by swiftyfinch** [comment]
Thank you! Looks fine to me. Also, let's add [the new job](https://github.com/swiftyfinch/Rugby/discussions/115#discussioncomment-1998278).

**Comment by devahmedshendy** [comment]
@swiftyfinch I added the job and the other points as mentioned [here](https://github.com/swiftyfinch/Rugby/discussions/115#discussioncomment-1999404).

**Review by swiftyfinch — APPROVED** [review]

File: .github/workflows/regress.yml [hunk 1]
```diff
@@ -19,25 +19,34 @@ jobs:
       - name: Check spelling
         run: sh .github/scripts/checkSpell.sh
 
-  rugby:
-    runs-on: macos-11.0
+  rugby-12.5.1:
+  runs-on: macos-11.0
```
Comment (swiftyfinch) [inline .github/workflows/regress.yml:23]: We need to shift this block, there is GitHub Action syntax error

**Review by swiftyfinch — CHANGES_REQUESTED** [review]
Please, fix job syntax

**Comment by devahmedshendy** [comment]
@swiftyfinch done

**Review by swiftyfinch — CHANGES_REQUESTED** [review]
Oh, also we need to rename jobs. I think maybe `rugby-12_5_1`.
> The workflow is not valid. .github/workflows/regress.yml (Line: 22, Col: 3): The identifier 'rugby-12.5.1' is invalid. IDs may only contain alphanumeric characters, '_', and '-'. IDs must start with a letter or '_' and and must be less than 100 characters. .github/workflows/regress.yml (Line: 39, Col: 3): The identifier 'rugby-12.4' is invalid. IDs may only contain alphanumeric characters, '_', and '-'. IDs must start with a letter or '_' and and must be less than 100 characters.

**Review by swiftyfinch — APPROVED** [review]

File: .github/workflows/regress.yml [hunk 2]
```diff
@@ -35,9 +35,18 @@ jobs:
         with:
           name: rugby
           path: ${{ env.rugby_path }}
+
+  rugby-12_4:
+    runs-on: macos-10.0
```
Comment (swiftyfinch) [inline .github/workflows/regress.yml:40]: Try to change this to `macos-10.15`
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#choosing-github-hosted-runners

> Can't find any online and idle self-hosted runner in the current repository, account/organization that matches the required labels: 'macos-10.0'
Waiting for a runner to pick up this job...

**Comment by swiftyfinch** [comment]
Everything looks fine to me. Let's merge it

**Comment by swiftyfinch** [comment]
@devahmedshendy Thanks for your contribution 🏈

**Pull request merged**